src/scicookie/{{cookiecutter.project_slug}}/pyproject.toml

```diff
@@ -51,6 +51,9 @@ pytest = "^7.3.2"
 pytest-cov = "^4.1.0"
 {% endif %}
 {%- endif -%}{#- end of use_pytest -#}
+{%- if cookiecutter.use_hypothesis == "yes" -%}
+hypothesis = "^6.0"
+{%- endif -%}
```

Comment:
```suggestion
{% endif -%}
```

Reply:
you can also use this to play with templates:

http://jinja.quantprogramming.com/